src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -1272,38 +1139,12 @@ public function testSubmitMultipleExpandedObjectChoices()
         $this->assertNull($form[4]->getViewData());
     }
 
-    public function testSubmitMultipleExpandedNumericChoices()
```

Comment:
if this test doesn't fail, it should not be removed. just rename `numericChoicesFlipped`